build.gradle

```diff
@@ -1,18 +1,25 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext {
+        detekt_version = "1.10.0"
+    }
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.2"
+        classpath "com.android.tools.build:gradle:4.2.2"
```

Comment:
@hoangnguyen92dn @lydiasama Should we reduce this version? 🤔

Reply:
Hmm. If we use 7.x version, the repository that doesn’t update the gradle version will unable to use our library, right?

Reply:
@lydiasama @doannimble Detekt builds failed with the latest gradle version 😄  That's why I had to downgrade the version.

Reply:
@hoangnguyen92dn Have you tried to update in `settings.gradle` ?
@lydiasama Correct, they have to upgrade to use.

Anw, just my wondering, I'm ok to downgrade the `gradle` so far 👌

Reply:
@doannimble 🤔 Do we need to update any configuration on `settings.gradle`? I think it's better support compatibility for project using lower gradle version rather than forcing them update to newer version.

Reply:
@doannimble 👋